source/unified-test-format/schema-1.3.json

```diff
@@ -213,23 +234,87 @@
           "properties": {
             "command": { "type": "object" },
             "commandName": { "type": "string" },
-            "databaseName": { "type": "string" }
+            "databaseName": { "type": "string" },
+            "hasServerId": { "type": "boolean" }
           }
         },
         "commandSucceededEvent": {
           "type": "object",
           "additionalProperties": false,
           "properties": {
             "reply": { "type": "object" },
-            "commandName": { "type": "string" }
+            "commandName": { "type": "string" },
+            "hasServerId": { "type": "boolean" }
           }
         },
         "commandFailedEvent": {
           "type": "object",
           "additionalProperties": false,
           "properties": {
-            "commandName": { "type": "string" }
+            "commandName": { "type": "string" },
+            "hasServerId": { "type": "boolean" }
+          }
+        },
+        "poolCreatedEvent": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {}
+        },
+        "poolReadyEvent": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {}
+        },
+        "poolClearedEvent": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "hasServerId": { "type": "boolean" }
+          }
+        },
+        "poolClosedEvent": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {}
+        },
+        "connectionCreatedEvent": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {}
+        },
+        "connectionReadyEvent": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {}
+        },
+        "connectionClosedEvent": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "reason": { "type": "string" }
+          }
+        },
+        "connectionCheckOutStartedEvent": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {}
+        },
+        "connectionCheckOutFailedEvent": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "reason": { "type": "string" }
           }
+        },
+        "connectionCheckedOutEvent": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {}
+        },
+        "connectionCheckedInEvent": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {}
```

Comment:
Related to my comment about `ignoreResultAndError` below, I think any schema changes should be accompanied by extra tests. See https://github.com/mongodb/specifications/commit/62c6bb9ed030e91cdab8622299bb1bf970b7f68d#diff-25ec19412b8ee6ccb0d93ebcd467ba7958e722164db42c5fbcc9e38de3ae2fd8.

This is tedious, but I think it's as worthwhile as spec tests themselves.

Reply:
As discussed in Slack, this can wait until other points in the PR are sorted out to reduce churn.

Reply:
Added tests to `valid-pass`, `valid-fail`, and `invalid` in my new commits.